controllers/deployment.go

```diff
@@ -43,12 +43,13 @@ func (d *DeploymentPlan) Init() {
 	// Default: each service runs in its own Pod
 	d.planMap[0] = ServiceGroups{
 		stateful: map[ServiceGroupName]ServiceGroup{
-			"logs":      {serviceLogs},
-			"messaging": {serviceMessaging},
-			"metrics":   {serviceMetrics},
-			"preview":   {servicePreview},
-			"appfabric": {serviceAppFabric},
-			"runtime":   {serviceRuntime},
+			"logs":          {serviceLogs},
+			"messaging":     {serviceMessaging},
+			"metrics":       {serviceMetrics},
+			"preview":       {servicePreview},
+			"appfabric":     {serviceAppFabric},
+			"runtime":       {serviceRuntime},
+			"supportbundle": {serviceSupportBundle},
```

Comment:
Why the support bundle service is a statefulset?

Reply:
customer will store their support bundle in local dir for now so we need storage